docs/cli/linear.rst

```diff
@@ -9,19 +70,32 @@ For training, use
 .. code-block:: bash
 
     python3 main.py --config CONFIG_PATH \
-                    --linear [--liblinear_options LIBLINEAR_OPTIONS]
+                    --linear \
+                    --liblinear_options LIBLINEAR_OPTIONS \
+                    --linear_technique MULTILABEL_TECHNIQUE
 
-- **config**: configure parameters in a yaml file.
+- **config**: configure parameters in a yaml file. Notice that the validation set is not needed to be specified additionally as the program will ignore it.
 
 The linear classifiers are based on LIBLINEAR, and its options may be specified.
 
+- **linear**: When this option exists, it is set to True to tell program that the linear classifier is used. Defaults to False.
```

Comment:
I don't think this gives sufficient reasons why this --linear is needed? What if users didn't specify it?